specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/textanalytics.json

```
@@ -0,0 +1,2254 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "Microsoft Cognitive Language Service",
+        "description": "The language service API is a suite of natural language processing (NLP) skills built with best-in-class Microsoft machine learning algorithms.  The API can be used to analyze unstructured text for tasks such as sentiment analysis, key phrase extraction, language detection and question answering. Further documentation can be found in <a href=\"https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/overview\">https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/overview</a>.0",
+        "version": "2021-11-01-preview"
+    },
+    "securityDefinitions": {
+        "apim_key": {
+            "type": "apiKey",
+            "name": "Ocp-Apim-Subscription-Key",
+            "in": "header"
```

You should include a description here so that there is some guidance to users of what to provide in the header.

---

Based on [this doc](https://docs.microsoft.com/en-us/azure/cognitive-services/authentication?tabs=powershell#authentication-headers):
```suggestion
            "in": "header",
            "description": "A subscription key for a specific service or a multi-service subscription key."
```

---

Updated!  Let me know if the language I used here works.